src/cli/run_command.zig

```diff
@@ -1142,6 +1147,8 @@ pub const RunCommand = struct {
                 .result => {},
             }
             fsinstance.top_level_dir = path;
```

Comment:
only this one should be necessary

Reply:
The `cwd` member does seem to be used in the `kind` function [here](https://github.com/oven-sh/bun/blob/26a720defe3a0962f8b1062734b221b838613d4a/src/fs.zig#L1252-L1270), but that's the only spot. The other places in the `fs` implementation just use `std.fs.cwd`. So I think to ensure that the `kind` of a file is correctly resolved this should be updated (but maybe I'm wrong on that, in general the `cwd` handling is kind of a mess)